lib/httparty/request.rb

```diff
@@ -201,7 +201,9 @@ def query_string(uri)
         query_string_parts << options[:query] unless options[:query].nil?
       end
 
-      query_string_parts.reject!(&:empty?) unless query_string_parts == [""]
+      query_string_parts.reject! { |part|
```

Comment:
I checked locally and this isn't needed to make any specs pass. Do we need a new spec or is this extra as well?